Pawfect-Harmony/build.gradle

```diff
@@ -6,10 +6,8 @@ plugins {
 
 group = 'org.launchcode'
 version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
 
-java {
-	sourceCompatibility = '11'
```

Comment:
Why was this changed?
I want to know out of curiosity.

Reply:
For a while I was having trouble getting the program to build because of a connection to the MySQL database. I ended up trying a few things in the build.gradle file due to the error message being about "implementation 'mysql:mysql-connector-java'. The error said it couldn't get the library. At one point, the suggestion I was looking at had the java{sourceCompatibility = '11' where I presently put it in build. gradle. I'm not sure it really helped though. In the end the change build.gradle needed was "implementation 'mysql:mysql-connector-java:8.0.15'". We could probably switch the java source compatibility back down in the build.gradle to it's original position. I just haven't tried putting it back in it's earlier place to see if still works

Reply:
Ok, I was just wondering.
I vote leave it alone.

Thank you for all of your hard work!